apps/mobile/src/features/approver/components/inputs-outputs-card.tsx

```diff
@@ -0,0 +1,41 @@
+import { t } from '@lingui/macro';
+
+import { Avatar, Box, Flag, ItemLayout, LockIcon, Text, UnlockIcon } from '@leather.io/ui/native';
+
+function UtxoRow({ isLocked }: { isLocked: boolean }) {
```

Comment:
How about just using the `<Flag` inline instead of having `UtxoRow`? 

Or else move it into it's own component